### DISCUSSION TITLE
Slack workspace

### DISCUSSION THREAD

**Discussion by aymanbagabas** [aymanbagabas/Huawei-WMI#82] [Announcements]
Feel free to join our Slack workspace at https://join.slack.com/t/slack-p3d4843/shared_invite/zt-266mn7bgk-tt49jNZZ~DHYribN4tpL0A

**Comment by VitalyVaryvdin** [comment]
Link seems to be dead

I've just got myself Matebook X Pro 2023 and would like to assist in development any way I could help.

**Reply by aymanbagabas** [reply]
Here's a new link https://join.slack.com/t/huawei-wmi/shared_invite/zt-2acst5t9f-~63bbsHH81LjHIrANRlq9w
